File: proxy_detection/java/Get.java
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;
import java.io.IOException;
import java.net.URI;

public class Get {
  public static void main(String[] args) throws IOException {
    HttpClient client = HttpClient.newHttpClient();
    HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://proxydetection.apifex.com/api/v1/detect"))
    .build();

    client.sendAsync(request, HttpResponse.BodyHandlers.ofString())
    .thenApply(HttpResponse::body)
    .thenAccept(System.out::println)
    .join();
  }
}

File: proxy_detection/nodejs/get.js
const https = require('https');

https.get('https://proxydetection.apifex.com/api/v1/detect?address=1.0.4.1', (resp) => {
  let data = '';

  resp.on('data', (chunk) => {
    data += chunk;
  });

  resp.on('end', () => {
    data = JSON.parse(data);
    console.log( 'The address ' + data.address + ' is a proxy: ' + data.is_proxy);
  });

});